services/telemetry/main.go

```diff
@@ -0,0 +1,232 @@
+package telemetry
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"runtime"
+	"strconv"
+	"time"
+
+	tsurumi "github.com/ironstar-io/telemetry"
+	"github.com/ironstar-io/tokaido/conf"
+	"github.com/ironstar-io/tokaido/system/version"
+	"github.com/ironstar-io/tokaido/utils"
+	uuid "github.com/satori/go.uuid"
+)
+
+// GenerateTelemetryID will generate a unique telemetry ID and save it to global config
+func GenerateTelemetryID() {
+	c := conf.GetConfig()
+	if c.Global.Telemetry.OptOut == true {
+		return
+	}
+
+	c.Global.Telemetry.Identifier = uuid.NewV4().String()
```

Comment:
Assuming you we're able to fix the method here?

Reply:
yeah, but I still have no idea why it was happening. I ended up just deleting my local copy and go getting again.